Variable fixes (#127)
citation-style-language/documentation: specification.rst

editorial-director
    Managing editor ("Directeur de la Publication" in French)

editor-translator

>     Automatically constructed if editor and translator are identical

Reviewer: I don't think this _has_ to be automatically constructed

```suggestion
    Editor and translator of a work. Can be automatically generated if editor and translator variables are identical
```

Author: Question is if this will ever be manually entered by users?

Author: But I wouldn't mind your suggestion.

Reviewer: Depends on the tool -- for people using CSL-YAML I could see that, also I think it's a variable in Biblatex (?) so in automated mapping from other formats. 
Maybe we should use `should` instead of `can` though if we want processors to behave consistently.

Author: Biblatex, no, I don't think so.
But I see your point with CSL-YAML. That could simplify things a bit.
Re `should`: I think it `must` be always automatically constructed if editor and translator are identical. That does not prevent users from just using the `editor-translator` variable directly in CSL-YAML.

Author: But I think we understand `should` in a rather prescriptive sense. So yeah, `should` is appropriate.

Reviewer: No, we follow very specific conventions for Must, should, etc as listed at the top of the specs under Terminology:
https://tools.ietf.org/html/rfc2119 
So if this should always happen, `must` is the right term. I've changed it above.

Author: Done

>     Editor and translator of a work. Must be automatically generated if editor and translator variables are identical

Reviewer 2: ```suggestion
    | Combined editor and translator of a work;
    | The citation processory must be automatically generate if ``editor`` and ``translator`` variables are identical;
    | May also be provided directly in item data
```

editorial-director
    Managing editor ("Directeur de la Publication" in French)

editor-translator
    | Combined editor and translator of a work;
    | The citation processory must be automatically generate if ``editor`` and ``translator`` variables are identical;
    | May also be provided directly in item data

executive-producer
    Executive producer (e.g. of a television series)

guest

>     Guest (e.g. of a TV show or podcast)

Reviewer 2: ```suggestion
    Guest (e.g. on a TV show or podcast)
```

executive-producer
    Executive producer (e.g. of a television series)

guest

>     Host

Reviewer: ```suggestion
    Guest (e.g. of a TV show or podcast)
```

host

>     Guest

Reviewer: ```suggestion
    Host (e.g. of a TV show or podcast)
```

reviewed-author
    Author of the item reviewed by the current item

script-writer
    Script-writer (e.g. of a movie)

> series-creator

Author: I think so, yes. But I'm not 100% sure.

> series-creator

Reviewer 2: Yes, “Series creator” often cited with that label for media research.

> series-creator

Author: Ok, I've added an example. Ok now?

reviewed-author
    Author of the item reviewed by the current item

script-writer

>     Script-writer (e.g. of a movie)

Reviewer 2: ```suggestion
    Writer of a script or screenplay (e.g. of a film)
```
